src/test/java/org/jenkinsci/plugins/docker/workflow/DockerDSLTest.java

```diff
@@ -149,7 +149,7 @@ private static void grep(File dir, String text, String prefix, Set<String> match
                 assumeDocker();
                 WorkflowJob p = story.j.jenkins.createProject(WorkflowJob.class, "p");
                 p.setDefinition(new CpsFlowDefinition(
-                        "docker.image('maven:latest').inside {\n" +
+                        "docker.image('maven:latest').overrideEntrypoint {\n" +
```

Comment:
Same problem exists with #78 - IMO, a worse problem, since I'd far prefer that the original behavior is reverted on `inside`.